### PR TITLE
Change product text

### PR DESCRIPTION
**Pull request by bilbof**

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ en:
             label: Individual
         custom_select_error: Select the best way to describe your business
       product_details:
-        title: Tell us about each product you’re offering
+        title: Tell us about the product you're offering
```
Comment (huwd) [inline config/locales/en.yml]:
```suggestion
        title: Tell us about the product you’re offering
```

I think we need to use the special `’` for these, no?
else they might clash with string closures?

right @issyl0 ?

**Review by huwd — COMMENTED** [review]
just that one backtick?

**Comment by bilbof** [comment]
@huwd great eyes!

**Review by huwd — APPROVED** [review]

**Comment by huwd** [comment]
No idea how to actually produce that character,
I keep having to go back to a slack convo with @issyl0 and copy pasting the last time it was mentioned :laughing:

**Comment by issyl0** [comment]
On a Mac, it's Alt+Shift+] to produce ’ (closing single curly quote, AKA an apostrophe).

**Pull request merged**